docs/learn/dapp-staking-v3/dapp-staking-faq.md

```diff
@@ -0,0 +1,82 @@
+Once v3 is launched, all of the unclaimed v2 rewards will become inaccessible. It doesn’t make sense to migrate rewards since the new [Tokenomics 2.0](/docs/learn/tokenomics2/) are tightly coupled with dApp staking - e.g. it’s not really possible to guarantee any inflation rate if we allow minting of over 2 year old rewards.
+
+To prevent users from loosing long accumulated rewards, a special state will be introduced into v2: **Decommission Mode**. Once decommissioning has started, majority of the dApps staking v2 functionality will be disabled, with the exception of reward claim calls. Two new special calls will also become available - _delegated staker reward claim_ & _delegated reward destination setting_.
+
+Using the new delegated calls, anyone will be able to claim rewards for anyone else. E.g. `Alice` can claim rewards for `Bob` - this essentially means that `Alice` pays for the transaction fee to claim `Bob's`. Of course, `Bob's` rewards are deposited into his account, not `Alice's` :slightly_smiling_face:.
+
+Astar team will ensure that all pending rewards are claimed during the decommissioning. We won’t launch v3 until all unclaimed rewards have been claimed.
+
+These transactions won’t be _free_, and whoever runs the _claim bot_ will need to be reimbursed by the treasury - but this is another topic. **TODO - please refer [here](https://forum.astar.network/t/dapp-staking-migration-from-v2-to-v3/5807) for more info.**
+
+Please do your best to claim the rewards yourself, and encourage others to do the same. This is just a one time thing, and won’t be something used again in dApp staking v3. Everyone is responsible for claiming their own rewards - also, with the new periods system, stakers will need to claim rewards & stake again roughly every 3 months to continue earning rewards.
+
+
+### Q: I am a Leger Astar Native App User, what do I need to do?
+
+:::danger
+For Ledger users who use the native account ([Astar Native App](https://support.ledger.com/hc/en-us/articles/10971402968733-Astar-ASTR)), dApp staking v3 will be temporarily unavailable.
+:::
+
+This is due to how Ledger app works & what is currently being developed.
+
+Ledger users will still be able to withdraw funds from dApp staking, they only won’t be able to stake & claim rewards in the new protocol version.
+
+**Why this limitation?**
+Ledger is currently developing a new generic app for all Polkadot/Kusama parachains, and that should be available soon (within months). Once that’s available, Ledger native users will be able to participate in dApp staking v3.
+
+**What can I do right now?**
+If you wish to actively participate in dApp Staking v3 right from the start, you should start the unboding period right now and move your funds to either Astar EVM Account supported by Ledger (see below) or alernatively move funds to a hot wallet.
+
+:::info 
+It is possible that by the launch of dApp staking v3, Ledger generic app will be launched, completely removing this limitation.
```

Comment:
Maybe also add something like "Keep an eye out on this docs and official announcements for more info as it becomes available"